Makefile

```diff
@@ -0,0 +1,23 @@
+# Using podman build --platform to cross compile. Podman will run non-native
+# platforms via `qemu-user-static` by default.
+
+all: third_party/bcachefs-tools/amd64/bcachefs third_party/bcachefs-tools/arm64/bcachefs third_party/bcachefs-tools/arm/bcachefs 
```

Comment:
Here and in *.go, please include the version in the path, i.e. `third_party/bcachefs-tools-1.7.0`